### ISSUE TITLE
I/O inside the event loop

### ISSUE THREAD

**Issue by whatdaybob** [custom-components/sensor.plex_recently_added#38]
**Describe the Bug**
I/O inside the event loop

**The Error**
```
2020-05-16 06:08:09 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for plex_recently_added doing I/O at custom_components/plex_recently_added/sensor.py, line 317: r = requests.head(url, verify=False)
```

**Indicate if you followed all steps in the [troubleshooting doc](https://github.com/custom-cards/upcoming-media-card/blob/master/troubleshooting.md)**
N/A

**What data does the sensor display if any?**
N/A

**What displays when visiting the API URL from the troubleshooting guide?**
N/A

**Does the default Home Assistant Plex component work for you?**
N/A

**Home Assistant Version**
0.109.3

**Additional Context**
Spams the logs on start up

**Comment by baldfox** [comment]
I too am having the same issue. Version: 0.113, 4.10 on hassio.... The sensor is working correctly but am still getting this error


Log Details (WARNING)
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 15:30:52 (20 occurrences)
Last logged: 15:30:58

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for plex_recently_added doing I/O at custom_components/plex_recently_added/sensor.py, line 323: r = requests.head(url, verify=False)

**Comment by maykar** [comment]
Should be fixed in latest release (0.3.3)
